doc/source/WorkingWithThePythonScriptingAPI/ThePythonEditor/index.md

```diff
@@ -0,0 +1,72 @@
+```eval_rst
+.. role:: raw-html(raw)
+    :format: html
+```
+
+# The Python Editor #
+
+The _Python Editor_ is a type of editor in the main Gaffer application that lets you build and modify the node graph, test expressions, and inspect data by inputting and executing Python code. In the default layout, it resides in the bottom-right panel, in a tab next to the _Hierarchy View_.
```

Comment:
If the audience potentially includes developers, it might be good to be a little more precise at the beginning here, how about:

_The _Python Editor_ is a type of editor in the main Gaffer application that lets you execute arbitrary python code within the UI's interpreter. This allows you to (amongst other things) build and modify the node graph and inspect data...._

> test expressions

Be careful here, the expression interpreter is not the same as the Python Editor's interpreter. As such, some things you can do in the Editor will not work in an expression and visa versa. For example, the expression engine defines additional `parent` and `context` variables in the global scope which have no equivalent in the Editor. If anything, its worth including a _warning_ here that there are differences between the two (and differences beyond those I mentioned here)!